shared/wallets/send-form/choose-asset/index.js

```diff
@@ -328,6 +328,9 @@ const styles = styleSheetCreate({
     isElectron: {
       maxHeight: 525 - 48,
     },
+    isMobile: {
+      paddingBottom: 100,
```

Comment:
i don't think this is likely the correct fix

Reply:
What do you have in mind?

Reply:
this kind of styling fix typically is fixing the symptom and not the actual problem. likely this list isn't setting its height correctly so its layout is wrong.